### PR TITLE
Issue 4469 - Backend redesign phase 3a (3rd PR)

### PR DESCRIPTION
**Pull request by progier389**
Followup of Prs about issue 4469
Fix some issues discovered after the review

### CODE REVIEW

File: ldap/servers/slapd/back-ldbm/db-bdb/bdb_layer.c [hunk 1]
```diff
@@ -6460,6 +6463,9 @@ int bdb_public_cursor_op(dbi_cursor_t *cursor,  dbi_op_t op, dbi_val_t *key, dbi
     DBT bdb_data = {0};
     int rc = 0;
 
+    if (bdb_cur == NULL)
```
Comment (Firstyear) [inline ldap/servers/slapd/back-ldbm/db-bdb/bdb_layer.c]: Braces around the return condition IE if (..) { return .... };

**Review by Firstyear — CHANGES_REQUESTED** [review]
Beside that, looks reasonable to me, but I'll let one of the others review too.

File: ldap/servers/slapd/back-ldbm/db-bdb/bdb_layer.c (hunk 1 above)
Comment (progier389) [reply ldap/servers/slapd/back-ldbm/db-bdb/bdb_layer.c]: ok. This change is now committed.

**Comment by progier389** [comment]
I run copr on the i4469 fork branch, and got success (except on one plateform that has a unrelated issue (nspr depemdemcy probem) So I am merging into upstream i4469 branch .

**Pull request merged**